_infra/helm/locust/locustfiles/locustfile.py

```diff
@@ -478,19 +478,31 @@ def post(self, url: str, data: dict = {}):
 class FrontstageTasks(TaskSet, Mixins):
 
     def on_start(self):
```

Comment:
on_start is called for each simulated respondent, we wouldn't want to load the json and assign to requests each time

Reply:
I have moved this into an __init__ function.

Reply:
Can you confirm if this comment re: init function is still valid? I can't see these changes and it looks ok as it currently is?

Reply:
I ended up moving the code that fetches the JSON file to the top. These comments were based on an earlier iteration of the PR where that code was in the FrontstageTasks part.